src/main/java/net/corevalue/multitenant/poc/config/MultiTenantProperties.java

```diff
@@ -24,6 +24,7 @@
     @Getter @Setter
     static class OrientDb {
```

Comment:
Make this @ ConfigurationProperties to enable config through a properties file

Reply:
It's already done, or probably I missed something?

Reply:
yes, PR doesn't show full code.